docs/containers.conf.5.md

```diff
@@ -424,6 +424,12 @@ The `engine` table contains configuration options used to set up container engin
 
 Name of destination for accessing the Podman service. See SERVICE DESTINATION TABLE below.
 
+**add-compression**=[]
+
+List of compression algorithms. If set makes sure that requested compression variant
+for each platform is added to the manifest list keeping original instance intact in
+the same manifest list on every `manifest push`. Supported values are (`gzip`, `zstd` and `zstd:chunked`)
```

Comment:
As someone who has not been looking at any of your work this description seems a bit short to me. What is the difference to the `compression_format` option? What happens when the original instance already contains the correct compression, I assume its a NOP in this case?

I think this needs more description and some cross references between these two options.

For example what happens if I have have compression_format="gzip" and add_compression=["zstd"], would the final manifest include both gzip and zstd?

Also maybe it is better to rename this to something like compression_manifest_formats? This would make the option be sorted next to the other compression options which makes it easier to find and follow for users IMO.

Reply:
> For example what happens if I have have compression_format="gzip" and add_compression=["zstd"], would the final manifest include both gzip and zstd?

@Luap99 Yes that is correct, now we can create multi-compression-variant manifest. I will extend documentation to explain this more.

Reply:
This question makes the CLI too complex.  We need to make this simple for the user, and my original goal was to allow containers.conf to specify zstd:chunked, gzip format, and hide the fact from the users.

Then if the user only wanted to deal with gzip, they could specify --compression gzip.